src/android/jsc_override.cpp

```diff
@@ -134,7 +134,7 @@ static JSGlobalContextRef create_context(JSContextGroupRef group, JSClassRef glo
     swap_function();
 
     // Clear cache from previous instances.
-    realm::_impl::RealmCoordinator::clear_all_caches();
+    RJSInvalidateCaches();
```

Comment:
For reference, here is this function: https://github.com/realm/realm-js/blob/5b0eabf940d843b76835b6e889817f4052c0b5c3/src/jsc/jsc_init.cpp#L57-L62